web/documentserver-example/php/webeditor-ajax.php

```diff
@@ -570,3 +574,52 @@ function renamefile()
 
     return ["result" => $commandRequest];
 }
+
+/**
+ * Reference data
+ *
+ * @return array
+ */
+function reference()
+{
+    $post = json_decode(file_get_contents('php://input'), true);
+
+    @header("Content-Type: application/json");
+
+    $referenceData = $post["referenceData"];
+
+    if ($referenceData) {
+        $instanceId = $referenceData["instanceId"];
+        if ($instanceId == serverPath()) {
+            $fileKey = json_decode($referenceData["fileKey"]);
+            $userAddress = $fileKey->userAddress;
+            if ($userAddress == getCurUserHostAddress()) {
+                $fileName = $fileKey->fileName;
+            }
+        }
+    }
+
+    if (!$fileName) {
```

Comment:
Add search for `fileName` by `path` like this
https://github.com/ONLYOFFICE/document-server-integration/commit/b27e532a830c2f8a6a3df0225bde487a0ec4a842#diff-7398df16c124c271406c203da02d406e5a6bc55926e9e224b52b0f68d99f4097R497-R503

Reply:
Search for `fileName` by `path` added + some possible exceptions in AJAX handled.